api/include/opentelemetry/plugin/tracer.h

```diff
@@ -50,9 +50,13 @@ class Span final : public trace::Span
 
   trace::Tracer &tracer() const noexcept override { return *tracer_; }
 
+  context::Token *GetToken() const noexcept override { return nullptr; }
```

Comment:
Curious, does this have to be public?

Reply:
I think it can be made private.

Reply:
Although if it is made private I don't see a reason for it to exist